test/e2e/cloud/imports.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cloud
+
+import (
+	// ensure these packages are scanned by ginkgo for e2e tests
+	_ "k8s.io/kubernetes/test/e2e/cloud/gcp"
```

Comment:
Do we have a tracking bug for getting rid of this as part of the cloud provider extraction effort?

Reply:
I'm not sure. It's not even clear to me these tests exercise the GCP cloud provider per se. 
 There are likely many cluster/kube-up.sh assumptions baked into these tests.

Reply:
AFAIK both integration and e2e lack a plan for this and fairly extensively pull in cloud provider